packages/web-components/src/switch/switch.ts

```diff
@@ -0,0 +1,15 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTSwitch } from '@microsoft/fast-foundation';
+import { SwitchLabelPosition } from './switch.options.js';
+
+export class Switch extends FASTSwitch {
+  /**
+   * The label position of the switch
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: label
+   */
+  @attr
+  public labelPosition: SwitchLabelPosition = 'after';
```

Comment:
This needs attribute syntax and we should likely keep this default (since it's presentational only) in CSS and not here.

```suggestion
  @attr({ attribute: "label-position" })
  public labelPosition: SwitchLabelPosition;
```